Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Jamulus/Startpage](https://hosted.weblate.org/projects/jamulus/1-index/).


It also includes the following components:

* [Jamulus/Installation-for-iOS](https://hosted.weblate.org/projects/jamulus/installation-for-ios/)

* [Jamulus/FAQ](https://hosted.weblate.org/projects/jamulus/faq/)

* [Jamulus/Server-Troubleshooting](https://hosted.weblate.org/projects/jamulus/server-troubleshooting/)

* [Jamulus/Include-Client-Commands](https://hosted.weblate.org/projects/jamulus/include-client-commands/)

* [Jamulus/Installation-for-Macintosh](https://hosted.weblate.org/projects/jamulus/installation-for-macintosh/)

* [Jamulus/Command-Line-Options](https://hosted.weblate.org/projects/jamulus/command-line-options/)

* [Jamulus/Include-Shared-Commands](https://hosted.weblate.org/projects/jamulus/include-shared-commands/)

* [Jamulus/Copyright notices](https://hosted.weblate.org/projects/jamulus/copyright-notices/)

* [Jamulus/General strings](https://hosted.weblate.org/projects/jamulus/general/)

* [Jamulus/Unregistered-Servers](https://hosted.weblate.org/projects/jamulus/unregistered-servers/)

* [Jamulus/Installation-for-Windows](https://hosted.weblate.org/projects/jamulus/installation-for-windows/)

* [Jamulus/Server-Bandwidth](https://hosted.weblate.org/projects/jamulus/server-bandwidth/)

* [Jamulus/Getting-Started](https://hosted.weblate.org/projects/jamulus/getting-started/)

* [Jamulus/Software-Manual](https://hosted.weblate.org/projects/jamulus/software-manual/)

* [Jamulus/QOS-Windows](https://hosted.weblate.org/projects/jamulus/qos-windows/)

* [Jamulus/Include-Backing-Up](https://hosted.weblate.org/projects/jamulus/include-backing-up/)

* [Jamulus/Running-a-Server](https://hosted.weblate.org/projects/jamulus/running-a-server/)

* [Jamulus/Installation-for-Android](https://hosted.weblate.org/projects/jamulus/installation-for-android/)

* [Jamulus/Client-Troubleshooting](https://hosted.weblate.org/projects/jamulus/client-troubleshooting/)

* [Jamulus/Privacy-Statement](https://hosted.weblate.org/projects/jamulus/privacy-statement/)

* [Jamulus/Contribution](https://hosted.weblate.org/projects/jamulus/contribution/)

* [Jamulus/Navigation](https://hosted.weblate.org/projects/jamulus/navigation/)

* [Jamulus/Tips-Tricks-More](https://hosted.weblate.org/projects/jamulus/tips-tricks-more/)

* [Jamulus/Installation-for-Linux](https://hosted.weblate.org/projects/jamulus/installation-for-linux/)

* [Jamulus/Directories](https://hosted.weblate.org/projects/jamulus/directories/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/jamulus/1-index/horizontal-auto.svg)

Commit:
Update Spanish translation using Weblate

Currently translated at 100.0% (6 of 6 strings)

Translation: Jamulus/Include-Client-Commands
Translate-URL: https://hosted.weblate.org/projects/jamulus/include-client-commands/es/

Commit:
Update Spanish translation using Weblate

Currently translated at 100.0% (74 of 74 strings)

Translation: Jamulus/Tips-Tricks-More
Translate-URL: https://hosted.weblate.org/projects/jamulus/tips-tricks-more/es/

pljones commented on _translator-files/po/es/Tips-Tricks-More.po:
```diff
@@ -350,57 +352,61 @@ msgid ""
+"     | letra de control | Control Jamulus |\n"
+"     |--------:|---------|\n"
+"     | `f` | Fader |\n"
+"     | `p` | Paneo   |\n"
```
No reason for the extra trailing spaces, is there?

ignotus666 replied:
Fixed.

trebmuh committed:
Update French translation using Weblate

Currently translated at 100.0% (6 of 6 strings)

Translation: Jamulus/Include-Client-Commands
Translate-URL: https://hosted.weblate.org/projects/jamulus/include-client-commands/fr/

Commit:
Update Spanish translation using Weblate

Currently translated at 100.0% (74 of 74 strings)

Translation: Jamulus/Tips-Tricks-More
Translate-URL: https://hosted.weblate.org/projects/jamulus/tips-tricks-more/es/
